files/mods.json

```diff
@@ -133,7 +133,7 @@
       "file": "optifine_1.8.9_HD_U_M5.jar",
       "url": "https://optifine.net/download?f=optifine_1.8.9_HD_U_M5.jar",
       "display": "optifine M5",
-      "description": "optifine allows your game to be more customizable and push it to its limits while also providing an FPS boost.",
+      "description": "Optifine is an EXTREMELY lucrative modification that is required for most mods and practically all SkyBlock texture packs. It also adds an incredible amount of customizability options for Minecraft's graphics settings, from tree leaves to fog.",
```

Comment:
What mods is optifine required for lol

Reply:
neu dwarven overlay is the only one i can think of

Reply:
optifine is needed to change textures depending on the name / nbt / other stuff